pkg/controller/backup_controller_test.go

```diff
@@ -231,6 +235,19 @@ func TestProcessBackup(t *testing.T) {
 				res.Status.Expiration.Time = expiration
 				res.Status.Phase = v1.BackupPhase(phase)
 
+				// We don't care about the value of the timestamps here, just whether or
```

Comment:
Is it not possible to use the value from the patch?

Reply:
It is - I can extract it and parse it to make the test more accurate. I was taking a bit of a shortcut here to avoid that work, but I'll update it.